modules/ROOT/pages/_partials/mobile-client-registration-procedure.adoc

```diff
@@ -4,9 +4,25 @@
 To create a {mobile-client}:
 
 . Log into the OpenShift console.
+<<<<<<< HEAD
+. Choose your project.
+. Click *Add to Project* and choose *Browse Catalog* from the options.
++
+You can filter the catalog items to only show mobile specific items by clicking the *Mobile* tab.
++
+image:catalog-all.png[Available Mobile Clients]
+. Click the *Apps* tab and then choose the mobile platform (Android, iOS, Cordova or Xamarin) and follow the wizard.
++
+NOTE: On the wizard's *configuration* screen, it is recommended that you input the name of the showcase application into the *Package Name* text box.
++
+NOTE: When registering an iOS app, you are prompted for a bundle id. See the link:https://cocoacasts.com/what-are-app-ids-and-bundle-identifiers/[description of bundle identifiers] for more information.
```

Comment:
I could be wrong but I am not sure we are supposed to provide links to 3rd party websites.  Its probably ok but need to get the policy on linking to 3rd party sites clarified.